app/logic/Mail/EWS/EWSAccount.ts

```diff
@@ -198,12 +201,58 @@ export class EWSAccount extends MailAccount {
     return options;
   }
 
+  async checkMaxConcurrency(ex: EWSItemError) {
+    if (ex.type != "ErrorServerBusy") {
+      throw ex;
+    }
+    // Check whether the ConcurrentSyncCalls policy was exceeded.
+    if (/'ConcurrentSyncCalls'/.test(ex.message) && /'(\d+)'/.test(ex.message) && Number(RegExp.$1) < this.maxConcurrency) {
```

Comment:
Can you make the second regexp more specific, please? To avoid that there's another `'4'` before `ConcurrentSyncCalls`. Like:

```suggestion
    if (/'ConcurrentSyncCalls'.*'(\d+)'/.test(ex.message) && Number(RegExp.$1) < this.maxConcurrency) {
```